Entities/Timeslot.cs

```diff
@@ -0,0 +1,13 @@
+using FRITeam.Swapify.Entities.Enums;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FRITeam.Swapify.Entities
+{
+    public class TimeSlot : BaseEntity
+    {
+        public eDay Day { get; set; }
+        public int StartHour { get; set; }
```

Comment:
int for number from 7-20?

Reply:
Yes is it neccessarry check if Starthour is from this range?

Reply:
I mean that byte is better :-)

Reply:
ok i changed it